Tools/generate-docc-reference/GenerateDoccReference.swift

```diff
@@ -50,6 +50,11 @@ struct GenerateDoccReference: ParsableCommand {
     help: "Directory to save generated docc reference. Use '-' for stdout.")
   var outputDirectory: String
 
+  @Option(
+    name: .shortAndLong,
+    help: "Use docc flavored markdown for the generated output.")
+  var doccFlavored: Bool = false
```

Comment:
I think I might prefer an option more like `--style <github|docc>`

Reply:
Easy change, I'll set it up.